paddle/framework/operator.h

```diff
@@ -142,9 +141,11 @@ class OperatorBase {
 // You can also use
 //   using PARENT_CLASS::PARENT_CLASS;
 // to use parent's constructor.
-#define DEFINE_OP_CONSTRUCTOR(CLS, PARENT_CLS)                                 \
-  CLS(const std::string& type, const VarNameMap& inputs,                       \
-      const VarNameMap& outputs, const paddle::framework::AttributeMap& attrs) \
+#define DEFINE_OP_CONSTRUCTOR(CLS, PARENT_CLS)             \
```

Comment:
I think `CLS` and `PARENT_CLS` should be lowercase.
See this example from Google c++ code style: https://google.github.io/styleguide/cppguide.html#Macro_Names

```cpp
#define ROUND(x) ...
```
`x` is lowercase.